src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -111,6 +111,9 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 	}
 
 	ngOnInit() {
+		this._register(DOM.addDisposableListener(this.toolbarElement.nativeElement, DOM.EventType.FOCUS_IN, () => {
```

Comment:
Why not put the listener in the component HTML instead? It'd be nice to be consistent where we can.

Reply:
I think we might need to add the listener to the textCell.component as well right?

Reply:
yeah I did that and it wasn't working as expected and there were no errors on the console either, so went back to adding the listener in the code.

Reply:
> I think we might need to add the listener to the textCell.component as well right?

@barbaravaldez no, since we added the focus listener on the HTML to call updateActiveCell.